base/dcp_sharded.go

```diff
@@ -439,8 +442,11 @@ func getMinNodeVersion(cfg cbgt.Cfg) (*ComparableVersion, error) {
 	return minVersion, nil
 }
 
-// StopHeartbeatListener unregisters the listener from the heartbeater, and stops it.
-func (c *CbgtContext) StopHeartbeatListener() {
+// Stop unregisters the listener from the heartbeater, and stops it and associated handlers.
+func (c *CbgtContext) Stop() {
```

Comment:
I agree it makes sense to have a more generic CbgtContext.Stop(), but if we're doing that, the rest of the 'on stop' code from importListener.Stop should get moved there as well.